### PR TITLE
feat: create_item

### PR DESCRIPTION
**Pull request by Jonathan-Atkins**
added create_item
updated routes

### CODE REVIEW

File: app/controllers/api/v1/items_controller.rb [hunk 1]
```diff
@@ -4,31 +4,41 @@ class Api::V1::ItemsController < ApplicationController
...
     items = Item.getItems(params)
     if items.is_a?(String)
-      render json: {"message": "your query could not be completed", "errors": ["#{items}"]}, status: 404
+      render json: { "message": "your query could not be completed", "errors": ["#{items}"] }, status: 404
     else
       render json: ItemSerializer.format_items(items)
     end
   end
 
+  def create
```
Comment (Jonathan-Atkins) [inline app/controllers/api/v1/items_controller.rb:13]: adding create

File: app/controllers/api/v1/items_controller.rb [hunk 2]
```diff
@@ -4,31 +4,41 @@ class Api::V1::ItemsController < ApplicationController
...
+      render json: { "message": "your query could not be completed", "errors": item.errors.full_messages }, status: 422
+    end
+  end
+
   def show
     item = Item.find(params[:id])
     render json: ItemSerializer.format_single_item(item)
   end
   
+  def destroy
```
Comment (Jonathan-Atkins) [inline app/controllers/api/v1/items_controller.rb:27]: my destroy was part of strong parameters. Moved it

File: app/controllers/api/v1/items_controller.rb [hunk 3]
```diff
@@ -4,31 +4,41 @@ class Api::V1::ItemsController < ApplicationController
...
+
   private
   
   def not_found_response(exception)
     render json: ErrorSerializer.format_error(exception, "404"), status: :not_found
-  rescue ActiveRecord::RecordNotFound
-    render json: { error: 'Item not found' }, status: :not_found
   end  
 
-  def destroy
```
Comment (Jonathan-Atkins) [inline app/controllers/api/v1/items_controller.rb:26]: wrong placement

**Review by wally-yawn — CHANGES_REQUESTED** [review]
Can you check that this isn't duplicating what Natasha put in in her PR? She had to do the create to get her update to work

**Pull request closed without merging**